### PR TITLE
feat: thrift-gen-http-swagger plugin

### PR DESCRIPTION
**Pull request by EZ4Jam1n**
#### What type of PR is this?

feat
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.
- [ ] Attach the PR updating the user documentation if the current PR requires user awareness at the usage level. [User docs repo](https://github.com/cloudwego/cloudwego.github.io)

#### (Optional) Translate the PR title into Chinese.
thrift-gen-http-swagger 插件

#### (Optional) More detailed description for this PR(en: English/zh: Chinese).

en:
zh(optional): 
有两个license没有放，为了合并的时候方便

### CODE REVIEW

File: thrift-gen-http-swagger/go.mod [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module github.com/hertz-contrib/swagger-generate/thrift-gen-http-swagger
+
+go 1.22
+
+require (
+	github.com/apache/thrift v0.13.0
+	github.com/cloudwego/hertz/cmd/hz v0.9.0
+	github.com/cloudwego/thriftgo v0.3.15
+	github.com/google/gnostic-models v0.6.8
```
Comment (li-jin-gou) [inline thrift-gen-http-swagger/go.mod:9]: 缺少 license

File: thrift-gen-http-swagger/go.mod [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	gopkg.in/yaml.v3 v3.0.1
```
Comment (li-jin-gou) [inline thrift-gen-http-swagger/go.mod:10]: 缺少 license

Comment (EZ4Jam1n) [reply thrift-gen-http-swagger/go.mod:10]: image
dd

File: thrift-gen-http-swagger/go.mod (hunk 1 above)
Comment (EZ4Jam1n) [reply thrift-gen-http-swagger/go.mod:9]: 这俩放pb那个pr里面了

**Review by li-jin-gou — APPROVED** [review]

**Pull request merged**